community/cfm/version

```diff
@@ -1 +1 @@
-0.4.1 3
+0.4.4 4
```

Comment:
`0.4.4 1`. You only bump the second field when you change the package's files but there as no package release.

Reply:
Could you elaborate on what this means? I'm not understanding properly. Fixed, however.

Reply:
- Software has an update -> First field (software version) is bumped.
- The KISS package files were changed (Possibly to change the `build` file or to fix a now invalid source as examples) -> Second field is bumped.

Reply:
Okay, thanks! Will keep that in mind.